### PR TITLE
1494 - Character issues paging not working

### PR DESCRIPTION
**Pull request by tomlovesgithub**
## What's included?
The pagination was incredibly buggy on the character reports page.
I was unable to tell exactly what this was intended to do as it wasn't functioning.
i believe the basis of the issue was the `orderBy(status)` which was causing results to be differently ordered upon a page change, null-ifying the change. 

## Who should test?
✅ Developers

## How to test?
- Find an exercise with more than 10 character issues
- go to exercise > reports > character issues
- test pagination (next + previous)

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

File: src/views/Exercise/Reports/CharacterIssues.vue [hunk 1]
```diff
@@ -284,9 +284,7 @@ export default {
       }
       if (this.candidateStatus !== 'all') {
         firestoreRef = firestoreRef.where('status', '==', this.candidateStatus);
-      } else {
-        firestoreRef = firestoreRef.where('status', '!=', 'withdrewApplication').orderBy('status');
-      }
+      }
```
Comment (warrensearle) [inline src/views/Exercise/Reports/CharacterIssues.vue:287]: I think you should just drop the `orderBy` from here but keep the `where` as this excludes withdrawn applications from the report.

**Review by warrensearle — DISMISSED** [review]
See comment about excluding withdrawn applications from the report.

**Review by HalcyonJAC — APPROVED** [review]
I tested using https://jac-apply-admin-production--pr1542-1494-paging-not-wor-xulbd33f.web.app/exercise/i1JUb4lIjGxAXcembR09/reports/character-issues# and the pagination seems to work fine.

File: src/views/Exercise/Reports/CharacterIssues.vue (hunk 1 above)
Comment (tomlovesgithub) [reply src/views/Exercise/Reports/CharacterIssues.vue:287]: removing only `orderby('status')` yeilds an error, i did try adding an additional orderby, but i couldnt work out the original ordering.

 "FirebaseError: [code=invalid-argument]: Invalid query. You have a where filter with an inequality (<, <=, >, or >=) on field 'status' and so you must also use 'status' as your first orderBy(), but your first orderBy() is on field 'candidate.fullName' instead."

**Comment by warrensearle** [comment]
@tomlovesgithub have made a few fixes to the paging

**Review by warrensearle — APPROVED** [review]
👍

**Pull request merged**